lightblock.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/wire"
+)
+
+type LightBlock struct {
+	height  int32
+	header  *wire.BlockHeader
+	lcStore Store
+}
+
+func (lb LightBlock) Height() int32 {
+	return lb.height
+}
```

Comment:
not needed, let's make height public

Reply:
We need make it public because we implement [HeaderCtx](https://github.com/btcsuite/btcd/blob/master/blockchain/interfaces.go#L41) from btcd packges.